public/modules/custom/grants_handler/src/Form/MessageForm.php

```diff
@@ -164,9 +165,76 @@ public function buildForm(array $form, FormStateInterface $form_state, WebformSu
       '#value' => $this->t('Send'),
     ];
 
+    $form_state->setStorage($storage);
+
     return $form;
   }
 
+  /**
+   * Validate & upload file attachment.
+   *
+   * @param array $element
+   *   Element tobe validated.
+   * @param \Drupal\Core\Form\FormStateInterface $formState
+   *   Form state.
+   * @param array $form
+   *   The form.
+   *
+   * @throws \Drupal\helfi_atv\AtvDocumentNotFoundException
+   * @throws \Drupal\helfi_atv\AtvFailedToConnectException
+   */
+  public static function validateUpload(array &$element, FormStateInterface $formState, array &$form) {
```

Comment:
Onko jotain syytä olla laittamatta tietoa funktion paluuarvosta?

```suggestion
  public static function validateUpload(array &$element, FormStateInterface $formState, array &$form): void {
```

Reply:
koska phpstorm ei sitä tarjonnu